docs/_tutorials/read-only-json-api.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Read-Only JSON API
+author: izdwuut
+date: 2020-12-23 10:04:00 +0100
+plugin_disclaimer: true
+---
+
+In this tutorial, you'll create a kind of a [REST](https://restfulapi.net/) [API](https://en.wikipedia.org/wiki/API) serving
+static [JSON](https://www.json.org/json-en.html) files generated from Markdown posts in your Jekyll site.
+
+JSON is a popular format to structure and exchange data on the web, without the presentation layer of HTML. The files you
+output are similar to a REST API in a server-side language like Ruby, Python, or Node. You can access them directly in your
+browser from e.g. `/foo.json`. They can be used to build a custom frontend using Angular, React, Vue, and so on.
+
+A huge inspiration to write this tutorial was a [post](https://forestry.io/blog/build-a-json-api-with-hugo/) written by
+[Régis Philibert](https://twitter.com/regisphilibert). It's about doing the exact same thing in [Hugo](https://gohugo.io/).
+
+You will need to define a number of [layouts]({% link _docs/layouts.md %}) and write a [plugin]({% link _docs/plugins.md %})
+to convert your content to JSON. **Don't be afraid if you're not familiar with Ruby** --- the language that Jekyll is
+written in --- it can be learned quite quickly.
+
+By the end of the tutorial, you'll expose the following API endpoints:
+* `/` --- an index of all posts
+* `/{url}` --- details of a post
+* `/categories` --- an index of categories
+* `/categories/{category}` --- an index of posts under a category
```

Comment:
```suggestion
* `/{url}/` --- details of a post
* `/categories/` --- an index of categories
* `/categories/{category}/` --- an index of posts under a category
```

since [AFAICT](https://stackoverflow.com/a/68562806/12916) you cannot get rid of the slash:

```console
$ curl -IL https://izdwuut.github.io/jekyll-json-api-tutorial/categories
HTTP/2 301 
server: GitHub.com
content-type: text/html
permissions-policy: interest-cohort=()
strict-transport-security: max-age=31556952
location: https://izdwuut.github.io/jekyll-json-api-tutorial/categories/
access-control-allow-origin: *
expires: Wed, 28 Jul 2021 15:46:05 GMT
cache-control: max-age=600
x-proxy-cache: MISS
x-github-request-id: A044:447C:5DE631:890EB7:61017965
accept-ranges: bytes
date: Wed, 28 Jul 2021 15:36:52 GMT
via: 1.1 varnish
age: 47
x-served-by: cache-fty21356-FTY
x-cache: HIT
x-cache-hits: 1
x-timer: S1627486612.323765,VS0,VE1
vary: Accept-Encoding
x-fastly-request-id: 18eb9201adfa28e4b3ee52de9c53090e57201a12
content-length: 162

HTTP/2 200 
server: GitHub.com
content-type: application/json; charset=utf-8
permissions-policy: interest-cohort=()
strict-transport-security: max-age=31556952
last-modified: Thu, 14 Jan 2021 11:45:10 GMT
access-control-allow-origin: *
etag: "60002ec6-15"
expires: Wed, 28 Jul 2021 15:45:47 GMT
cache-control: max-age=600
x-proxy-cache: MISS
x-github-request-id: 5514:447C:5DE3FF:890B33:61017953
accept-ranges: bytes
date: Wed, 28 Jul 2021 15:36:52 GMT
via: 1.1 varnish
age: 65
x-served-by: cache-fty21356-FTY
x-cache: HIT
x-cache-hits: 1
x-timer: S1627486612.347649,VS0,VE1
vary: Accept-Encoding
x-fastly-request-id: e56fa6c0ce7c97e157f7c7ab45c59c92e7fed259
content-length: 21
```